doc/12-icinga2-api.md

```diff
@@ -2502,6 +2503,680 @@ curl -k -s -S -i -u root:icinga -H 'Accept: application/json' \
+      <size from="81" to="81" total="243" count="3"/>
+      <size from="97" to="97" total="679" count="7"/>
+      <size from="113" to="113" total="1130" count="10"/>
+      <size from="129" to="129" total="129" count="1"/>
+      <size from="161" to="161" total="322" count="2"/>
+      <size from="177" to="177" total="1416" count="8"/>
+      <size from="193" to="193" total="579" count="3"/>
+      <size from="209" to="209" total="627" count="3"/>
+      <size from="225" to="225" total="225" count="1"/>
+      <size from="257" to="257" total="1285" count="5"/>
+      <size from="289" to="289" total="289" count="1"/>
+      <size from="337" to="337" total="674" count="2"/>
+      <size from="353" to="353" total="1059" count="3"/>
+      <size from="369" to="369" total="369" count="1"/>
+      <size from="401" to="401" total="401" count="1"/>
+      <size from="417" to="417" total="417" count="1"/>
+      <size from="433" to="433" total="433" count="1"/>
+      <size from="465" to="465" total="465" count="1"/>
+      <size from="481" to="481" total="962" count="2"/>
+      <size from="497" to="497" total="497" count="1"/>
+      <size from="513" to="513" total="513" count="1"/>
+      <size from="529" to="529" total="1058" count="2"/>
+      <size from="545" to="545" total="1090" count="2"/>
+      <size from="657" to="657" total="657" count="1"/>
+    </sizes>
+    <total type="fast" count="0" size="0"/>
+    <total type="rest" count="82" size="50977"/>
+    <system type="current" size="344064"/>
+    <system type="max" size="344064"/>
+    <aspace type="total" size="344064"/>
+    <aspace type="mprotect" size="344064"/>
+    <aspace type="subheaps" size="1"/>
+  </heap>
+  <heap nr="30">
+    <sizes>
+      <size from="17" to="32" total="96" count="3"/>
+      <size from="33" to="48" total="576" count="12"/>
+      <size from="49" to="64" total="64" count="1"/>
+      <size from="97" to="112" total="3584" count="32"/>
+      <size from="49" to="49" total="98" count="2"/>
+      <size from="81" to="81" total="810" count="10"/>
+      <size from="257" to="257" total="2827" count="11"/>
+      <size from="689" to="689" total="689" count="1"/>
+      <size from="705" to="705" total="705" count="1"/>
+      <unsorted from="81" to="81" total="81" count="1"/>
+    </sizes>
+    <total type="fast" count="48" size="4320"/>
+    <total type="rest" count="27" size="118618"/>
+    <system type="current" size="135168"/>
+    <system type="max" size="135168"/>
+    <aspace type="total" size="135168"/>
+    <aspace type="mprotect" size="135168"/>
+    <aspace type="subheaps" size="1"/>
+  </heap>
+  <total type="fast" count="938" size="79392"/>
+  <total type="rest" count="700" size="4409469"/>
+  <total type="mmap" count="0" size="0"/>
+  <system type="current" size="15114240"/>
+  <system type="max" size="15114240"/>
+  <aspace type="total" size="15114240"/>
+  <aspace type="mprotect" size="15114240"/>
+</malloc>
+```
```

Comment:
This is very long and provides very little information for its length. Please shorten it to a reasonable length. I'd suggest replacing all but the first `<heap>` element with `<!-- ... -->`.